scene/gui/subviewport_container.cpp

```diff
@@ -223,6 +223,22 @@ void SubViewportContainer::unhandled_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+TypedArray<String> SubViewportContainer::get_configuration_warnings() const {
+	TypedArray<String> warnings = Node::get_configuration_warnings();
+
+	bool has_viewport = false;
+	for (int i = 0; i < get_child_count(); i++) {
+		if (get_child(i)->is_class("SubViewport")) {
+			has_viewport = true;
+		}
+	}
+	if (!has_viewport) {
+		warnings.push_back(TTR("This node doesn't have a SubViewport as child, so it can't display it's intended content.\nConsider adding a SubViewport as a child to provide something displayable."));
```

Comment:
```suggestion
		warnings.push_back(TTR("This node doesn't have a SubViewport as child, so it can't display its intended content.\nConsider adding a SubViewport as a child to provide something displayable."));
```